src/main/drivers/opticalflow/opticalflow.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include "common/time.h"
+
+#define OPTICALFLOW_OUT_OF_RANGE        (-1)
```

Comment:
(BF style- no unnecessary parentheses. But don't take it too literally, sometimes code is easier to read with them)
```suggestion
#define OPTICALFLOW_OUT_OF_RANGE        -1
```